src/tasks/level10.ts

```diff
@@ -97,7 +99,8 @@ export const GiantQuest: Quest = {
       completed: () => have($item`amulet of extreme plot significance`),
       do: $location`The Penultimate Fantasy Airship`,
       choices: () => {
-        return { 178: 2, 182: have($item`model airship`) ? 1 : 4, 1387: 3 };
+        // eslint-disable-next-line libram/verify-constants
+        return { 178: 2, 182: have($item`bat wings`) ? 6 : have($item`model airship`) ? 1 : 4, 1387: 3 };
```

Comment:
In your testing, does this actually cause it to use choice 6 appropriately for you? In my own (separate) experimentation, I needed to modify the choice adventure script to get it to select 6 when up but apply other logic when not.

Reply:
It sure doesn't! And I changed the choice adventure script similarly because of it.

Reply:
Also, if you have any changes you want to push to this PR, happy to collaborate on it. Better than going solo.